lib/LaTeXML/Package/aas_support.sty.ltxml

```diff
@@ -65,6 +74,9 @@ DefMacro('\slugcomment{}', '\@add@frontmatter{ltx:note}[role=slugcomment]{#1}');
 
 DefMacro('\shorttitle{}',   '\@add@frontmatter{ltx:toctitle}{#1}');
 DefMacro('\shortauthors{}', '');                                      # not useful?
+# Meant to be used redundantly with an \author macro, so no metadata to deposit in this call.
+DefMacro('\correspondingauthor{}', '\footnote{Corresponding author: #1}');
```

Comment:
A bit better would be
```
\lx@contact{correspondent}{#1}
```
(similar to elsart) which would preserve it as metadata.

Reply:
Sounds good! I was wondering whether I should preserve the human-visible footnote as well, but happy to go for the metadata-pure route.

Reply:
Switching to the macro you offered, the XML now contains:
```xml
  <creator>
    <contact role="correspondent">Yjan A. Gordon</contact>
    <contact role="email">yjan.gordon@umanitoba.ca</contact>
  </creator>
```

but, curiously, when converting to HTML, that entry is entirely dropped without any messages? I think the `creator` element would need some role attribute to get preserved...

Reply:
In any case, that would be a separate issue altogether -- looking at the `1905.00018` example I mentioned, there seem to be a handful of stylistic updates needed to get the paper back to a correct  and stylized title page look.

I've committed the macro change, and am recording the current paper heading for information:
![title](https://user-images.githubusercontent.com/348975/60021527-2ac39b80-9660-11e9-96fc-e4609138e684.png)

Respective HTML snippet here:
```html
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document">
<h1 class="ltx_title ltx_title_document">THE EFFECT OF MINOR AND MAJOR MERGERS ON THE EVOLUTION OF 
<br class="ltx_break">LOW EXCITATION RADIO GALAXIES</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">[
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">Department of Physics and Astronomy, University of Manitoba,
Winnipeg, MB R3T 2N2, Canada
</span>
<span class="ltx_contact ltx_role_affiliation">E.A. Milne Centre for Astrophysics, University of Hull, Cottingham Road,
Kingston-upon-Hull HU6 7RX, U.K.
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Kevin A. Pimbblet
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">E.A. Milne Centre for Astrophysics, University of Hull, Cottingham Road,
Kingston-upon-Hull HU6 7RX, U.K.
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Sugata Kaviraj
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">Centre for Astrophysics Research, School of Physics, Astronomy and Mathematics,
University of Hertfordshire, 
<br class="ltx_break">College Lane, Hatfield AL10 9AB, U.K.
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Matt S. Owers
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">Department of Physics and Astronomy, Macquarie University, Sydney, NSW 2109, Australia
</span>
<span class="ltx_contact ltx_role_affiliation">Astronomy, Astrophysics and Astrophotonics Research Centre, Macquarie University, Sydney, Australia
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Christopher P. O’Dea
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">Department of Physics and Astronomy, University of Manitoba,
Winnipeg, MB R3T 2N2, Canada
</span>
<span class="ltx_contact ltx_role_affiliation">School of Physics and Astronomy, Rochester Institute of Technology,
84 Lomb Memorial Drive, Rochester, NY 14623, USA
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Mike Walmsley
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">Department of Physics, University of Oxford, Denys Wilkinson Building,
Keble Road, Oxford OX1 3RH, U.K.
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Stefi A. Baum
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">Department of Physics and Astronomy, University of Manitoba,
Winnipeg, MB R3T 2N2, Canada
</span>
<span class="ltx_contact ltx_role_affiliation">School of Physics and Astronomy, Rochester Institute of Technology,
84 Lomb Memorial Drive, Rochester, NY 14623, USA
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Jacob P. Crossett
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">School of Physics and Astronomy, University of Birmingham,
Edgbaston, Birmingham B15 2TT, U.K.
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">[
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">School of Physics and Astronomy, University of Nottingham,
University Park, Nottingham NG7 2RD, U.K.
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Chris J. Lintott
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">Department of Physics, University of Oxford, Denys Wilkinson Building,
Keble Road, Oxford OX1 3RH, U.K.
</span></span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Jonathon C.S. Pierce
</span><span class="ltx_author_notes"><span>
<span class="ltx_contact ltx_role_affiliation">Department of Physics and Astronomy, University of Sheffield, Sheffield S3 7RH, U.K.
</span></span></span></span>
</div>
<div class="ltx_date ltx_role_received">November 23, 2018March 25, 2019April 30, 2019</div>
<div class="ltx_date ltx_role_revised">November 23, 2018March 25, 2019April 30, 2019</div>
<div class="ltx_date ltx_role_accepted">November 23, 2018March 25, 2019April 30, 2019</div>
```

Reply:
Hmm, yes, you're right; it does disappear. There's no obvious standard translation of such metadata into html (other than some rdf?);  Perhaps the best approach would be to do both: record the metadata and create a footnote (or whatever way a particular style presents "corresponding author")?

Reply:
I would definitely be in favor of adding publisher metadata in standard RDFa eventually. But since we don't have a clear target ("Scholarly HTML"? Any other dialect?) for RDFa at the moment, it may be best to punt on that until 0.8.5. It may be a good forward-facing discussion to have in e.g. #896 .

For this PR, I think it makes most sense to just stick to the macro you suggested (effectively a no-op for the HTML), as a placeholder for RDFa to come. Feel free to merge if that seems reasonable for now.